### ISSUE TITLE
Clean up and re-organize repo

### ISSUE THREAD

**Issue by mechalynx** [snintendog/Youtube-Player-Editing-2015#8]
The repository is a mess. I can't tell which versions of the .css files are intended for what and if they're improvements over one another. The Dev directory is also confusing. I tried diffing the files and reading the commits to make some sense of it but it's too much work.

I wanted to update the userscript and automate the inclusion of the css but at this state I don't know what to include and what to get rid of. If we are to keep various testing versions of files, it's best to keep them in a separate development branch instead of master. Git keeps track of changes so there's no need to redundantly comment statements out, it just makes it confusing.

Please merge the changes so we know what's a test, what the latest working version is and use shorter names. If there is a need for more description, add it as a comment at the beginning of the file.

As per the norm for repos, I propose we keep release files in a directory called `dist` (where the final userscript and/or userstyles recide, those that are to be distributed to end users), keep the sources files in a directory called `src` (which we can combine with a script automatically to produce those that go in `dist`) and have a separate branch called `testing` or `dev` for development, from which changed are merged back to `master` when ready, without breaking things.

The root should only have the readme, license and a gruntfile, if we want to automate the merging of css and js into a single file so users don't need to get multiple ones to get this to work.

edit - I forked the project and cleaned up the Readme and some files. If any of the changes I made are good for inclusion, tell me and I'll make a separate branch with those changes and make a pull request.

**Comment by snintendog** [comment]
Yea I can completely agree that i have made a mess of this repo sorry

**Comment by snintendog** [comment]
Started to deleting some of the outdated stuff on there added the folders and yeah you already did better than me at reorganizing on your fork once again sorry for the confusion and mess.

**Comment by mechalynx** [comment]
It's fine, don't take it so hard. This is clearly just a lack of experience. The rule of thumb is to keep things clear enough so even someone who's never seen this code before can understand what everything is so they can work on it. My primary gripe was with the multiple versions of the same css co-existing, which seems to be fixed. I might adjust some things for clarity later, just to keep them consistent with standards - feel free to roll back whatever you don't like.

Since the problem was primarily the css files, this issue is fixed as far as I'm concerned, as long as you're done clearing them up.

**Comment by snintendog** [comment]
Yeah those were mostly remnants from when i was trying to figure out a more universal way of getting the player to scale correctly. I will Try to keep it as clean as possible from now on. You really should see the mess that my local folder looks like cleaning that one up next  I had made a new one for every minor change to the css paranoid I had made a mistake and broke everything in the previous version. My lack of experience is definitely showing.

**Comment by mechalynx** [comment]
Well, nobody's born with experience :P. If you want to keep older versions just in case, either make the changes on github and let it do its revisioning magic, or just use git locally, clone the repo and it keeps a history of commits. This kind of paranoia is why revisioning exists in the first place, healthy for coders :P

**Comment by snintendog** [comment]
yeah....oops didn't even know of that feature on github thought i just kept track of the changes nothing else load off my shoulders if anything. Also bringing your version of the read me over Really well done on that  and professional looking.
